### PR TITLE
Fixess #667 - Broadcast Page Error

### PR DESCRIPTION
**Pull request by Anurag-Singh-Rajput**
## Issue that this pull request solves

 Closes: #667

#### **CREATED UNDER SWOC 2.0 PROGRAM**
Broadcast page wasn't able to fetch data and show it on screen, on second reload page became white, this issue is fixed.

- The deployed backend wasn't able to return the data from any of the routes. So, for now I have changed the END_POINT in .env file in /frontend to localhost so that it doesn't create issues to other users because the backend code will change. 
- useEffect was called only once when the page loaded, so I added the array "dataa" as the dependancy.
**Conclusion** - Issue#667 has been  fixed

## Types of changes

_Put an `x` in the boxes that apply_

- [x ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_API="https://hitk-community-website-backend.herokuapp.com"
+REACT_APP_API="http://localhost:3500"
```
Comment (Kajol-Kumari) [inline frontend/.env:1]: Hey @Anurag-Singh-Rajput the backend of this website is hosted at this location only so can you please change back to the heroku one. It should work fine.

Comment (Anurag-Singh-Rajput) [reply frontend/.env:1]: Ok @Kajol-Kumari , Actually the backend doesn't seem to return any json data when i tried the route https://hitk-community-website-backend.herokuapp.com/broadcast 
 I will look into it and make the necessary changes.

Comment (Anurag-Singh-Rajput) [reply frontend/.env:1]: Hey @Kajol-Kumari ,
I think I found the problem. I tried to deploy same backend code to heroku and i was able to access /broadcast route.
I think there might be one of these problems, please check on Heroku(where backend is deployed and probably connected to GitHub):
1.) Is the Auto-deploy turned on?
2.) Is the branch selected in deploy is "Master"?

Comment (Kajol-Kumari) [reply frontend/.env:1]: Thanks for pointing this @Anurag-Singh-Rajput I checked the hosted-backend and I found issue with the auto-deploy, I fixed that and now the bug seems to be resolved. 

As you can see the data isn't getting populated on the cards of the broadcast page, can you look into the same?

**Review by Kajol-Kumari — APPROVED** [review]
LGTM

**Comment by Kajol-Kumari** [comment]
@Anurag-Singh-Rajput Could you please mention in the PR description under which program this PR has been made?

File: frontend/.env (hunk 1 above)
Comment (Anurag-Singh-Rajput) [reply frontend/.env]: @Kajol-Kumari  Should I add program name under "Brief Description of what is Fixed or changed ' ?

Comment (Kajol-Kumari) [reply frontend/.env]: Yes, this works we just need to know under which program this PR has been created in order to keep the track :)

Comment (Anurag-Singh-Rajput) [reply frontend/.env]: Great Thanks ! This was my first PR :)

**Pull request merged**